### ISSUE TITLE
Rescan folder using MediaScanner after state flips from syncing to 100% idle

### ISSUE THREAD

**Issue by Catfriend1** [Catfriend1/syncthing-android#626]
### Description of the issue
https://github.com/syncthing/syncthing-android/issues/1488

We need some pretty small and effective code with good error catching to do the following:
- If a folder state transitions from "syncing" to "idle 100%" check if .nomedia files are present in the folder
- If one or more of them are present, rescan the folder with MediaScanner.

**Comment by bell07** [comment]
My proposal is to remember during sync if any ".nomedia" file was added, changed or deleted from folder. If they was such change, trigger the full rescan of the folder with changed file after the sync was completed.

**Comment by Catfriend1** [comment]
Yes, the idea is good, but I currently can't make up my mind for a light implementation.

**Comment by Catfriend1** [comment]
This is now possible to do after we detected a folder sync to be complete:
https://github.com/Catfriend1/syncthing-android/pull/1406
